lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -16314,3 +16314,123 @@ msgstr "Souple"
 
 msgid "User is not allowed to reach web application"
 msgstr "L'utilisateur n'est pas autorisé à joindre l'application web"
+
+msgid "Define the password security policy"
+msgstr "Définir la politique de sécurité de mot de passe"
+
+msgid "Password case policy"
+msgstr "Politique de casse du mot de passe"
+
+msgid "Password minimum length"
+msgstr "Longueur minimum du mot de passe"
+
+msgid "Minimum 8 characters"
+msgstr "8 caractères minimum"
+
+msgid "Maximum 128 characters"
+msgstr "128 caractères maximum"
+
+msgid "Force to use upper case"
+msgstr "Forcer à utiliser une majuscule"
+
+msgid "Force to use lower case"
+msgstr "Forcer à utiliser une minuscule"
+
+msgid "Force to use numbers"
+msgstr "Forcer à utiliser des nombres"
+
+msgid "Force to use special characters"
+msgstr "Forcer à utiliser des caractères spéciaux"
+
+msgid "Password cases"
+msgstr "Casses du mot de passe"
+
+msgid "Weak"
+msgstr "Faible"
+
+msgid "Good"
+msgstr "Bon"
+
+msgid "Strong"
+msgstr "Fort"
+
+msgid "Password expiration policy"
+msgstr "Politique d'expiration du mot de passe"
+
+msgid "Password expiration"
+msgstr "Expiration du mot de passe"
+
+msgid "Choose a duration between 1 hour and 12 months"
+msgstr "Veuillez choisir une durée entre 1 heure et 12 mois"
+
+msgid "Month"
+msgstr "Mois"
+
+msgid "Months"
+msgstr "Mois"
+
+msgid "Day"
+msgstr "Jour"
+
+msgid "Days"
+msgstr "Jours"
+
+msgid "Hour"
+msgstr "Heure"
+
+msgid "Hours"
+msgstr "Heures"
+
+msgid "Minute"
+msgstr "Minute"
+
+msgid "Minutes"
+msgstr "Minutes"
+
+msgid "Second"
+msgstr "Seconde"
+
+msgid "Second"
```

Comment:
`Seconds` is already translated

Reply:
But we need the singular here

Reply:
yes but you have a dupicate `msgid "Second"` :)